hackathon_site/dashboard/frontend/src/pages/Inventory/Inventory.test.tsx

```diff
@@ -34,13 +35,48 @@ describe("Inventory Page", () => {
             .calledWith(categoriesUri)
             .mockResolvedValue(categoryApiResponse);
 
-        const { getByText } = render(<Inventory />);
+        const { getByText, queryByText } = render(<Inventory />);
 
         await waitFor(() => {
             expect(getByText(mockHardware[0].name)).toBeInTheDocument();
             expect(getByText(mockCategories[0].name)).toBeInTheDocument();
         });
 
+        // by default the hardwareUri has a limit of 100
+        if (mockHardware.length <= 100) {
+            expect(queryByText(/load more/i)).not.toBeInTheDocument();
+        } else {
+            expect(getByText(/load more/i)).toBeInTheDocument();
+        }
+    });
+
+    test("Hardware count", async () => {
+        // Mock hardware api response with a limit of mockHardware.length - 2
+        const limit = mockHardware.length - 2;
+        const hardwareApiResponse = makeMockApiListResponse(
+            mockHardware.slice(0, limit),
+            null,
+            null,
+            mockHardware.length
+        );
+
+        when(mockedGet)
```

Comment:
Since you're rendering the full inventory page, you should also mock the categories response that it expects. Right now it will just pass a bunch of mock objects through which won't break anything, but in theory they could if we ever incorporate more error checking.